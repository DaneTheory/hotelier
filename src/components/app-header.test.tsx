import React from 'react';
import renderer from 'react-test-renderer';
import AppHeader from './app-header';

describe('AppHeader', () => {
  it('matches the snapshot', () => {
    const rendered = renderer.create(<AppHeader/>).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
