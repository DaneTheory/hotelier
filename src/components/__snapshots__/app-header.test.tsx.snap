// Jest Snapshot v1, https://goo.gl/fbAQLP

exports['AppHeader matches the snapshot 1'] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-primary"
>
  <div
    className="mx-auto font-weight-bold text-center text-white"
  >
    Hotelier
  </div>
</nav>
`;
